js/validation.js

```diff
@@ -0,0 +1,42 @@
+function validate()
+{
+	//document.body.appendChild(form);
+	
+	console.log("eho");
+	
+	let username = document.getElementById("username").value;
+	let usernamePattern = /^[a-zA-Z0-9_]{3,10}$/;
+	
+	let pass = document.getElementById("pass").value;
+	let passPattern = /^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9]).{6,}$/;
+	
+	let pass2 = document.getElementById("pass2").value;
+	
+	if (! validateField(username, usernamePattern))
+	{
+		document.write("<div>Username " + username + " not correct. ");
```

Comment:
You may want to consider not using document.write. More information: https://stackoverflow.com/a/4537982/3837638